src/_guides/libraries/c-interop.md

```diff
@@ -90,11 +90,13 @@ The [`hello.dart` file]({{ page.hw}}/hello.dart)
 illustrates the steps for using dart:ffi to call a C function:
 
 1. Import dart:ffi.
-2. Create a typedef with the FFI type signature of the C function.
-3. Create a typedef for the variable that you'll use when calling the C function.
-4. Open the dynamic library that contains the C function.
-5. Get a reference to the C function, and put it into a variable.
-6. Call the C function.
+2. Import the path library that you'll use to store the path of dynamic library.
```

Comment:
If importing Platform, Directory, & path is really necessary, we should probably put that in the first step:

1. Import the dart:ffi library and APIs to construct the library path.

If we don't want to unconditionally recommend it (@dcharkes?), that makes the writing more complicated, but we should preface this with _Optional:_:

2. _Optional:_ Import APIs to construct the library path.

Reply:
@kwalrath, I am not sure if these APIs are really necessary. Though the example mentioned in the tutorial uses these to define the path variable for the dynamic library, but I could get the same result by passing the relative file address directly to the function like this:
```dart
final dylib = ffi.DynamicLibrary.open('hello_library/libhello.so');
```